.github/workflows/create_draft_release.yml

```diff
@@ -14,17 +14,11 @@
 name: Create Draft Release
 
 on:
-  workflow_dispatch: # input version manually. Overrides push tag
-    inputs:
-      tag:
-        description: "Release version, eg:latest, 0.2.1"
-        required: true
-        default: "0.0.0"
+  workflow_dispatch: # select tag when creating
```

Comment:
```suggestion
  on:
     release:
         types: [published]
```

use this instead?

Reply:
What would be the intended use case? We have an action further down in the script ( `softprops/action-gh-release@v2`) to create a release, and that becomes quite useless if we already have created a release.

And adding `on:release` is a bit superfluous. If you create a new release, and the tag does not exist since before a new tag will be created and that will anyway trigger this workflow.

I am actually considering if we should remove `workflow_dispatch` - I do not see that many use-cases for it, except possibly if you have yanked a release and want to rebuild a new release for the same tag.